Update ddev instructions in marketplace doc (#16804)
DataDog/documentation: content/en/developers/marketplace/offering.md

## Install and configure the Datadog development toolkit

The Agent Integration Developer Tool allows you to create scaffolding when you are developing an integration by generating a skeleton of your integration tile's assets and metadata. For instructions on installing the tool, see [Install the Datadog Agent Integration Developer Tool][25].

> Once you've installed the Developer tool, configure the tool for the `marketplace` repo:

Reviewer: ```suggestion
After you install the Developer tool, configure it for the `marketplace` repo:
```
